src/clarity/forms/_forms.clarity.scss

```diff
@@ -258,34 +270,39 @@ $clr-custom-checkbox-radio-top: ($clr_baselineRem_1 - $clr-checkbox-radio-height
                 padding-left: 0;
                 margin-bottom: $clr_baselineRem_1;
 
-                & > label:first-of-type {
-                    position: relative;
+                & > label:first-child,
+                & > label:not(:first-child),
```

Comment:
hmm. not loving it. i will be happy(er) when we can refactor to a CSS arch that is easier to maintain.